src/utils/hydration.js

```diff
@@ -142,19 +142,85 @@ function injectRepeatedRows({ step, repeatedRowCount }) {
   };
 }
 
+const formatTrackValue = (value, type = 'col', isEmpty) => {
+  if (isEmpty && value === 'auto') value = '0'; // Default empty value
+  if (type === 'col') return `minmax(0, ${value})`;
+  return value;
+};
+
+const formatDimensionValue = (value, type = 'col') => {
+  switch (value) {
+    case 'fill':
+      return type === 'col' ? '100%' : 'fit-content';
```

Comment:
can you include a comment here about why fit-content for rows too? and why fit-content for the auto case too?